app/scripts/directives/bindService.js

```diff
@@ -242,7 +268,13 @@
       };
 
       var serviceClass = BindingService.getServiceClassForInstance(svcToBind, ctrl.serviceClasses);
-      BindingService.bindService(svcToBind, application, serviceClass, ctrl.parameterData).then(function(binding){
+
```

Comment:
I don't know if other kinds of services would cause this function to bloat over time, but it may be wise to branch early:

```JavaScript 
// call separate functions early, which can share another function if necessary to deduplicate.
ctrl.bindService = function() {
   if (ctrl.isMobileEnabled && isMobileService(serviceClass)) {
      bindMobileService();  
   } else {
      bindService();
   }
}
```

Reply:
:+1: